internal/app/lifecycle.go

```diff
@@ -90,7 +90,7 @@ func (l *Lifecycle) OnStopped() func() {
 		return *stopped
 	}
 
-	if *stopped == nil {
```

Comment:
Is is possible for `stopped` to be non-nil but `*stopped` to be still nil? Should the check be `if stopped == nil || *stopped == nil`?

Reply:
Yes, it is possible someone stored nil good catch. Fixed